staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -31,6 +31,8 @@ import (
 	"github.com/coreos/etcd/clientv3"
 	"k8s.io/klog"
 
+	utiltrace "k8s.io/utils/trace"
```

Comment:
nit: previously it was sorted alphabetically, so I think it made more sense previously